modules/manage/pages/manage-nodes/modify-services-and-rebalance.adoc

```diff
@@ -0,0 +1,170 @@
+= Modify Services and Rebalance
+:description: pass:q[Add or remove non-Data Services on existing nodes in a cluster and rebalance the cluster without adding or removing nodes.]
+:page-toclevels: 3
+
+[abstract]
+{description}
+
+Dynamically add or remove the following Multi-Dimensional Scaling (MDS) services on existing nodes in a cluster from the Couchbase xref:manage:manage-nodes/modify-services-on-nodes-and-rebalance.adoc#modify-mds-services-from-ui[UI], xref:rest-api:rest-set-up-services-existing-nodes.adoc[REST API], or xref:manage:manage-nodes/modify-services-on-nodes-and-rebalance.adoc#modify-mds-services-using-cli[CLI]:
+
+* `index` (Index Service)
+* `n1ql` (Query Service)
+* `fts` (Search Service)
+* `cbas` (Analytics Service)
+* `eventing` (Eventing Service)
+* `backup` (Backup Service)
+
+Then a rebalance operation is automatically triggered to distribute the service workload across the nodes and complete the modification.
+
+You do not have to add or remove a node to add or remove a non-Data Service on a node in a cluster.
+
+NOTE: You cannot add or remove kv (Data Service) using this method.
+
+Adding or removing of the Data Service on an existing node is supported only through adding or removing nodes.
+
+To add a Data Service on an existing node of a cluster from the UI:
+
+. Remove the node that you want to add the Data Service to using xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+. Then xref:manage:manage-nodes/add-node-and-rebalance.html[add the node] back to the cluster along with enabling the Data Service.
++
+Alternatively, you can add a new node with the necessary services configuration, including the Data Service, using xref:manage:manage-nodes/add-node-and-rebalance.adoc[Add a New Node and Rebalance].
+Then remove the extraneous node that you no longer need using xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+
+To remove a Data Service from an existing node of a cluster using the UI:
+
+. Remove the node with the Data Service that you want to remove using xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
+. Then xref:manage:manage-nodes/add-node-and-rebalance.html[add the node] back to the cluster along with disabling the Data Service.
++
+During the node removal operation, if there are other services on the node that you're removing for the Data Service reconfiguration, and you do not want those services to be impacted when the node is removed from the cluster, you may need to add those services to another node temporarily.
+Then add a new node temporarily with those services enabled.
```

Comment:
Remove lines 25-39.   It's too much info.   All you need to say is (the rest are obvious details and depends on the customer's scenario):

To add or remove a Data sService on an existing node of a cluster:
. xref:manage:manage-nodes/remove-node-and-rebalance.adoc[Remove a Node and Rebalance].
. Then xref:manage:manage-nodes/add-node-and-rebalance.html[add the node] back to the cluster along with adding or removing the data service.